### PR TITLE
AK: Two tiny changes to make it compile on windoze

### PR DESCRIPTION
**Pull request by alimpfard**
Except Time.cpp, which is not needed for Jakt so I don't care :P

### CODE REVIEW

File: AK/Platform.h [hunk 1]
```diff
@@ -168,16 +168,20 @@
 #endif
 
 #ifndef AK_OS_SERENITY
+#    ifdef AK_OS_WINDOWS
+// FIXME: No idea where to get this, but it's probably 4096 anyway :^)
```
Comment (AtkinsSJ) [inline AK/Platform.h:172]: Pretty much. https://devblogs.microsoft.com/oldnewthing/20210510-00/?p=105200

Comment (alimpfard) [reply AK/Platform.h:172]: will remove the "probably"

File: AK/Platform.h [hunk 2]
```diff
@@ -168,16 +168,20 @@
...
+#    elif defined(AK_OS_MACOS)
 extern "C" {
 #    endif
 #    include <unistd.h>
 #    undef PAGE_SIZE
 #    define PAGE_SIZE sysconf(_SC_PAGESIZE)
 #    ifdef AK_OS_MACOS
-};
+}
+;
```
Comment (linusg) [inline AK/Platform.h:184]: :thonk:

Comment (alimpfard) [reply AK/Platform.h:184]: :yakshrug: I didn't even touch it
will fix

**Pull request merged**